mod.ts

```diff
@@ -1,3 +1,8 @@
+// Check for minimum supported V8 runtime version
+if(Number(Deno.version.v8.split(".")[0]) < 12) {
```

Comment:
```suggestion
if(Deno.version.deno >= 1.38) {
```
This might work fine.

Reply:
shouldnt it be `<=` ?